src/transformers/models/dpr/modeling_tf_dpr.py

```diff
@@ -82,7 +82,7 @@ class TFDPRContextEncoderOutput(ModelOutput):
             heads.
     """
 
-    pooler_output: tf.Tensor
+    pooler_output: tf.Tensor = None
```

Comment:
@LysandreJik @sgugger - for some reason, these outputs need to be initialized with a value or else our much-loved  tests `test_compile_tf_model` and `test_saved_model_with_...` fail. Tbh, I don't know why this is - seems to be some weird tf graph problem with positional arguments...-> @sgugger is this fine for you?

Reply:
Yes, I remember that now. Had to do the same for all tf ModelOutput. This is fine indeed!

Reply:
Great catch! :-)